### ISSUE TITLE
Laravel 11

### ISSUE THREAD

**Issue by 120dev** [owen-it/laravel-auditing#910]
| Q                 | A
| ----------------- | ---
| Bug?              | no
| New Feature?      | no
| Framework         | Laravel 
| Framework version | 11
| Package version   | 13.x.y
| PHP version       | 8.3.3

#### Actual Behaviour

Problem installing package with Laravel 11

#### Steps to Reproduce

`composer require owen-it/laravel-auditing
`
Make sure to **include code** (affected models, configuration, ...).

![image](https://github.com/owen-it/laravel-auditing/assets/4286701/2ce0a55d-4182-4501-b92d-9102d31fbbba)

In a nutshell, help us, help you.

**Comment by parallels999** [comment]
Duplicate of #907
